package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "andela-societies-frontend",
+  "version": "1.0.0",
+  "description": "Andela societies is an application that allows fellows to log and track points for their societies",
+  "main": "server/server.js",
+  "scripts": {
+    "start": "webpack-dev-server --config=webpack.config.dev.js --open --history-api-fallback",
+    "build": "webpack -p --config=webpack.config.prod.js && npm run copy:favicon",
+    "serve": "node server.js",
+    "test": "export NODE_ENV=testing && jest --verbose",
+    "test:dev": "export NODE_ENV=testing && jest --verbose --watch",
+    "copy:favicon": "cp src/assets/images/favicon.ico public/favicon.ico",
+    "deploy": "npm run build && npm run serve",
+    "coverage": "export NODE_ENV=testing && jest --coverage",
+    "coveralls": "cat ./coverage/lcov.info | coveralls"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AndelaOSP/andela-societies-frontend.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AndelaOSP/andela-societies-frontend/issues"
+  },
+  "homepage": "https://github.com/AndelaOSP/andela-societies-frontend#readme",
+  "devDependencies": {
+    "babel-eslint": "^8.2.1",
+    "eslint": "^4.18.2",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-plugin-import": "^2.9.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.7.0",
+    "mock-local-storage": "^1.0.5",
+    "webpack-dev-server": "^2.11.2"
+  },
+  "dependencies": {
```

Comment:
Can we move all dev dependencies to  ... devDependencies 😄 i.e `enzyme` ` coveralls`....

Reply:
I could do that